src/TaskPlanner/Contracts/Requests/PerformSwitchoverRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Middleware.TaskPlanner.Contracts.Requests;
+
+public record PerformSwitchoverRequest
+{
+    public Guid ActionPlanId { get; init; }
+    public Guid InstanceId { get; init; }
+    public string Destination { get; init; }
```

Comment:
Is Destionation attribute the name of the cloud or edge?

Reply:
yes